packages/flutter/test/cupertino/nav_bar_test.dart

```diff
@@ -4,7 +4,7 @@
 
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/rendering.dart';
-import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter_test/flutter_test.dart' hide TypeMatcher;
```

Comment:
why `hide TypeMatcher`?

Reply:
There's 2. The other one (that I'm using) is in widgets/frameworks